src/tests.builds

```diff
@@ -2,12 +2,17 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <Project Include="*\test*\**\*.csproj">
+    <Project Include="*\test*\**\*.csproj" Condition="'$(OSEnvironment)'=='Windows_NT'">
       <OSGroup Condition="'$(FilterToOSGroup)'!=''">$(FilterToOSGroup)</OSGroup>
     </Project>
     <Project Include="*\test*\**\*.vbproj" Condition="'$(IncludeVbProjects)'!='false'">
       <OSGroup Condition="'$(FilterToOSGroup)'!=''">$(FilterToOSGroup)</OSGroup>
     </Project>
+
+    <!-- Temp Workaround to unblock non-windows builds -->
+    <Project Include="*\test*\**\*.csproj" Exclude="**\Microsoft.VisualBasic.Tests.csproj" Condition="'$(OSEnvironment)'!='Windows_NT'">
```

Comment:
No need to separate these like this. Instead just create an ExcludeProjects itemgroup and put this test project in it based on the OS condition.